python/private/py_binary_rule.bzl

```diff
@@ -50,7 +33,6 @@ def create_binary_rule_builder():
         executable = True,
     )
     builder.attrs.update(AGNOSTIC_BINARY_ATTRS)
-    builder.attrs.update(_COVERAGE_ATTRS)
```

Comment:
Good to remove it from here, having to add these attrs is really hard as a user.